src/utils.ts

```diff
@@ -131,22 +135,22 @@ export function getAnimeQueries(name: string): string[] {
 		animeQueries.push(cleanseSeparators(`${title} ${release}`));
 	}
 	if (altTitle) {
-		// Edge cases from regex
-		if (altTitle.toLowerCase() === "season") return animeQueries;
-		if (altTitle.toLowerCase() === "ep") return animeQueries;
-
+		if (isBadTitle(altTitle)) return animeQueries;
 		animeQueries.push(cleanseSeparators(`${altTitle} ${release}`));
 	}
 	return animeQueries;
 }
 
-export function reformatTitleForSearching(name: string): string {
-	// use lazy regex evaluation
-	const fullMatch =
-		name.match(EP_REGEX)?.[0] ??
-		name.match(SEASON_REGEX)?.[0] ??
-		name.match(MOVIE_REGEX)?.[0] ??
-		name;
+export function reformatTitleForSearching(
+	name: string,
+	useRegex = true,
```

Comment:
I wanted to keep these here so that logic is not duplicated. Any changes we make here for non id searching should also apply to rss, announce, and ensemble.

Reply:
We can keep the logic in one place with a separate function too, by calling it inside this one.